<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWinClass</class>
 <widget class="QMainWindow" name="MainWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>udp4all</string>
  </property>
  <property name="windowIcon">
   <iconset resource="udp4all.qrc">
    <normaloff>:/images/images/udp4all.png</normaloff>:/images/images/udp4all.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Datagrams:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDatagrams">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>258</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionReconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionMonitor"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="udp4all.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="udp4all.qrc">
     <normaloff>:/images/images/fileopen.png</normaloff>:/images/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="udp4all.qrc">
     <normaloff>:/images/images/filesave.png</normaloff>:/images/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Monitor</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About udp4all</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="icon">
    <iconset resource="udp4all.qrc">
     <normaloff>:/images/images/network-connect.png</normaloff>:/images/images/network-connect.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;connect</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="udp4all.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWinClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
